### PR TITLE
FB 4156; Fix GoTo text field focus; other GoTo tweaks

### PR DESCRIPTION
**Pull request by zfox23**
Fixes:
- Forward/back button icon y-alignment
- Forward/back button hover/click visuals and behavior
- Text field auto-focus in Desktop mode
- GoTo show/hide/toggle behavior fixed
- Fixed warnings in console
- Fixed visual bugs around text field in desktop mode
- Added padding to text input field to be more in-line with UIT
- Cleaned up help text

*Test Plan:*
Precondition: Start up in home (so that there are no other places in your session history). Start up in Desktop mode.

1. In Desktop mode, open the GoTo address bar.
a. The input field should have keyboard focus: normal cursor (if any).
b. The input field should tell you where you are.
2. Enter some text using your physical keyboard.
a. The gray ""Your Location"" text should go away and your text should appear.
b. As you type each letter (or the delete key), the character should appear in the text field (or the previous letter should disappear if deleting).
c. As you type each letter (or the delete key), the suggestions should be re-filtered (on each key).
3. Click with the mouse off of the address bar (with text still entered)
a. The input field should lose focus.
4. Switch to HMD. In HMD, tap with the stylus/finger on the text input field. Then, input some text using the virtual keyboard
a. The input field should have keyboard focus: normal cursor
b. The virtual keyboard should appear, and the virtual keyboard should be the same size as elsewhere in the application.
c. Same results as in (2).
7. Switch to Desktop. Click the home button in the GoTo address bar.
a. The address bar should close.
b. The GoTo button should become unhighlighted.
8. Open the GoTo address bar.
a. The back and forward buttons should be disabled - visually and in fact.
9. Go someplace else (i.e. dev-welcome). Re-open the GoTo menu.
a. The back button should be enabled visually.
10. Click the back button
a. You should be back home
b. The location field should reflect that you are home.
c. The forward button should be enabled visually.
11. Click the forward button.
a. You should be at the same place as in (9)
b. The location field should reflect that.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 054370908130562edd617978fdf71a7338da0f3d available at: https://gist.github.com/6c60b3a8939ea4f006c2828dd3c7220d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-0543709.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-0543709.dmg

File: interface/resources/qml/hifi/tablet/TabletAddressDialog.qml [hunk 1]
```diff
@@ -21,7 +21,7 @@ import "../../styles-uit" as HifiStyles
 import "../../controls-uit" as HifiControls
 
 StackView {
-    id: root
+    id: root;
```
Comment (davidkelly) [inline interface/resources/qml/hifi/tablet/TabletAddressDialog.qml]: I thought no semicolons?  So -- match what is there currently?

**Comment by sharsein** [comment]
Testing

**Comment by sharsein** [comment]
Desktop PC - Windows 10 Home - CPU 3.5GHz Intel i5-6600K - GTX1060 - RAM 16GB - Tested with HTC Vive
Tested Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-0543709.exe

QA Failed: Functionally everything works. However the forward and back buttons are not visually disabled and will still highlight when mouse hovers over them.

**Comment by hifi-gustavo** [comment]
Build console output for commit: a623ced64ea8f76dbde1465bdd2e2f8bdbbec271 available at: https://gist.github.com/6c60b3a8939ea4f006c2828dd3c7220d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-a623ced.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-a623ced.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: a4587e33f331d9fa2112acba0b860efae85f8a83 available at: https://gist.github.com/6c60b3a8939ea4f006c2828dd3c7220d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-a4587e3.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-a4587e3.dmg

**Comment by zfox23** [comment]
QA ready once again, @sharsein. All that's changed is the button look and feel. Thanks for your help!

**Comment by sharsein** [comment]
Everything looks good. QA Pass

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7a7cfb84e57e6d364a38f3502b879592d84e2ab0 available at: https://gist.github.com/6c60b3a8939ea4f006c2828dd3c7220d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-7a7cfb8.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-7a7cfb8.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e6edce53524577b9e89cf972684eabcef7b3d8ac available at: https://gist.github.com/6c60b3a8939ea4f006c2828dd3c7220d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-e6edce5.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10213-e6edce5.dmg

**Pull request merged**